Makefile

```diff
@@ -106,3 +106,17 @@ GOBIN=$(PROJECT_DIR)/bin go get $(2) ;\
 rm -rf $$TMP_DIR ;\
 }
 endef
+
+
+
+CLUSTERNAME= civo-capi-provider
+down:
+	kind delete clusters $(CLUSTERNAME)
+
+up: down
+  kind create cluster $(CLUSTERNAME)\
+kind get kubeconfig > /tmp/kubeconfig && export KUBECONFIG=/tmp/kubeconfig\
```

Comment:
Let's add uniform identation everywhere. Otherwise `make` might not work: `	`.

Also, add `&&` at the end of each line(so before next command is executed). That would ensure that next command is run only when previous is succeded.

Reply:
will do 👍🏽